compiler/resolution/preFold.cpp

```diff
@@ -239,6 +239,14 @@ static Expr* preFoldPrimOp(CallExpr* call) {
 
   case PRIM_GATHER_TESTS: {
     int  totalTest = 0;
+    if (call->numActuals() == 0) {
+      USR_FATAL(call, "illegal call of 'gather tests'");
```

Comment:
I'd add a little bit more to this error message, like ", expected an argument of type Test"

Reply:
I have made the changes as suggested. Thanks